src/main/java/io/minio/Digest.java

```diff
@@ -95,7 +101,7 @@ public static String md5Hash(byte[] data, int length) throws NoSuchAlgorithmExce
    * @param len          Length of Input stream.
    */
   public static String md5Hash(Object inputStream, int len)
-    throws IllegalArgumentException, NoSuchAlgorithmException, IOException, InsufficientDataException {
+    throws NoSuchAlgorithmException, IOException, InsufficientDataException {
```

Comment:
is `IllegalArugumentException` under https://dev.eclipse.org/sonar/rules/show/squid:RedundantThrowsDeclarationCheck?